README.md

```diff
@@ -1 +1,83 @@
-# itn-installer
\ No newline at end of file
+# ITN Installer
+
+This repository contains an easy to use installer to run a Dusk Network node for our ITN program.
+
+## Prerequisites
+
+- Ubuntu 22.10 x64
+- OpenSSL 3
+
+This installer is specifically built for Ubuntu 22.10 x64. It might work on older or new version.
+
+## Packages
+
+The installer comes with the following packages:
+- Dusk node service
+- Rusk VM service
+- Rusk wallet CLI
+
+## Folder layout 
+
+The configuration files, binaries, services and scripts can be found in `/opt/dusk/`. 
+
+The log files can be found in `/var/logs/{d,r}usk.{err,log}`.
+
+## Installation
+
+To run the ITN installer execute the following command:
+```sh
+curl --proto '=https' --tlsv1.2 -sSfL https://github.com/dusk-network/itn-installer/releases/download/v0.0.3/itn-installer.sh | sudo sh
+```
+
+## Configuration
+
+The installer comes with sane defaults, only requiring minimal configuration. Before the Dusk and Rusk service can be started, the `CONSENSUS_KEYS` and `DUSK_CONSENSUS_KEYS_PASS` need to be provided. 
+
+The `CONSENSUS_KEYS` can be either moved to `/opt/dusk/conf/` from another system or generated on the node itself and moved there. 
+
+### Set consensus keys
+
+To generate the provisioner keys locally, run `rusk-wallet` and either create a new wallet or use a recovery phrase. Access your wallet and export your provisioner key-pair. 
+
+The `.keys` file will be appended with the wallet address you have selected. Move this file to the Dusk node configuration folder and rename it:
+```sh
+mv /root/.dusk/rusk-wallet/<ADDRESS>.keys /opt/dusk/conf/consensus.keys
+```
+
+### Set consensus password
+
+Set the password by running the following command:
+```sh
+echo 'DUSK_CONSENSUS_KEYS_PASS=<PROVISIONER_KEY_PASSWORD>' > /opt/dusk/services/dusk.conf
```

Comment:
Lot of people don't remove the `<>` placeholder

I believe that adding and example would be useful

Reply:
Done!